### DISCUSSION TITLE
where are the code make xaml to html/css in uno

### DISCUSSION THREAD

**Discussion by srxqds** [unoplatform/uno#8440] [Show and tell]
where are the code make xaml to html/css in uno and how can we write custom control

**Comment by jeromelaban** [comment]
Thanks for the question :) 

There's no specific location for the code that generates XAML to HTML, as each individual control handles its own mapping individually depending on how it's mapping features to HTML or CSS.

Now, your talking about a custom control and is it about about integrating some existing javascript control ([see this tutorial](https://platform.uno/docs/articles/interop/wasm-javascript-1.html)) or is it about creating your own custom controls using XAML?
